app/models/course/survey.rb

```diff
@@ -30,6 +30,15 @@ class Course::Survey < ApplicationRecord
     unscoped.joins(:lesson_plan_item).select(Course::LessonPlan::Item.arel_table[:id])
   end)
 
+  calculated :student_responses_count, (lambda do
```

Comment:
Maybe just a nit: `student_submitted_responses_count` since this scope is just for submitted responses